src/api/api.js

```diff
@@ -25,24 +26,77 @@ async function createEvent(event) {
 
   let res;
   try {
-    res = await protectedResourceAxios.post(`${process.env.VUE_APP_API_DOMAIN}/api/v1/protected/events/`, body);
+    res = await protectedResourceAxios.post('/api/v1/protected/events/', body);
   } catch (err) {
     return err;
   }
 
   return res;
 }
 
+const stripePromise = loadStripe(process.env.VUE_APP_STRIPE_PUBLISHABLE_KEY);
+
+async function handleClickCheckout(cartEvents, userLevel) {
+  if (userLevel === 'ADMIN' || userLevel === 'PF') {
+    const body = {
+      lineItems: cartEvents.map(event => ({
+        id: event.id,
+        name: event.name,
+        description: event.description,
+        amount: event.price,
+        currency: 'usd',
+        quantity: 1,
```

Comment:
Do we maintain a cartEvent item for each ticket a user buys? Because otherwise this number won't always be 1

Reply:
Right now the user can't choose a quantity as far as I know. This can be changed easily though. The backend will be configured to have a quantity.